custom_components/anna/manifest.json

```
@@ -0,0 +1,8 @@
+{
+  "domain": "anna",
+  "name": "Plugwise_Anna",
+  "documentation": "https://github.com/laetificat/anna-ha",
+  "dependencies": ["climate"],
```

Having this set to "climate" will cause my HASS instance to not start, currently running version 0.92.0, removing this will fix it.
```suggestion
  "dependencies": [],
```

---

Didn't push this (apologies) - for some reason when upgrading to 0.92.1 yesterday it did work, then break for me because of this.